app/server/runtime/src/test/java/io/syndesis/server/runtime/SetupITCase.java

```diff
@@ -15,32 +15,24 @@
  */
 package io.syndesis.server.runtime;
 
-import java.util.ArrayList;
-import java.util.List;
-
-import static java.util.concurrent.TimeUnit.SECONDS;
-
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-
 import io.syndesis.common.model.ListResult;
 import io.syndesis.common.model.connection.ConfigurationProperty;
 import io.syndesis.common.model.connection.Connection;
 import io.syndesis.common.model.connection.Connector;
-import io.syndesis.server.credential.CredentialFlowState;
-import io.syndesis.server.credential.CredentialProvider;
-import io.syndesis.server.credential.CredentialProviderLocator;
-import io.syndesis.server.credential.Credentials;
-import io.syndesis.server.credential.OAuth1CredentialFlowState;
-import io.syndesis.server.credential.OAuth1CredentialProvider;
+import io.syndesis.server.credential.*;
```

Comment:
I think in general we do always use full qualified imports.

Reply:
Ah ok, missed that one, thanks. As checkstyle was reporting in other files too. So I configured my IDE to use qualified imports but obviously was missing this one